docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,533 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d299c1e8",
+   "metadata": {},
+   "source": [
+    "# Finding Label Errors in Object Detection Datasets\n",
+    "\n",
+    "This 5-minute quickstart tutorial demonstrates how to find potential label errors in object detection datasets.  In object detection data, each image is annotated with multiple bounding boxes.  Each bounding box surrounds a physical object within an image scene, and is annotated with a given class label. Using this labeled data, we train a model to predict the locations and classes of objects in an image. Predictions from the trained model can subsequently be input to cleanlab in order to identify mislabeled images and a quality score quantifying our confidence in the overall annotations for each image.  After correcting these label issues, you can train an even better version of your model without changing your training code!\n",
+    "\n",
+    "An [example](https://github.com/cleanlab/examples) notebook to train the object detection model whose predictions we rely on in this tutorial is available [here](https://github.com/cleanlab/examples/blob/master/object_detection/detectron2_training.ipynb). Note that this tutorial just considers find label issues in a subset of the validation dataset based on held-out predictions for this subset. This tutorial can also be used to find all mislabeled images across an entire dataset, but you will want to first execute the [other example notebook](https://github.com/cleanlab/examples/blob/master/object_detection/detectron2_training-kfold.ipynb) that uses K-fold cross-validation to produce out-of-sample predictions for every image.\n",
```

Comment:
Suggestion to move this paragraph much lower in the tutorial (perhaps in section 2 where we explain that we loaded the data from a model trained earlier), or perhaps even deleting completely if it does not fit in well.

I don't think it should be here, as it confused me a little when I read it. This is also the introduction paragraph to using object detection in cleanlab, and imo how we got the data for this specific tutorial is not the most important detail. And esp if it will potentially confuse readers I think we should move it.

Also, currently the COCO dataset is introduces AFTER this paragraph, so users at this point will have no idea what we mean by "this dataset" or "this subset".

Also some suggested text edits:
```suggestion
    "An [example](https://github.com/cleanlab/examples) notebook to train the object detection model whose predictions we rely on in this tutorial is available [here](https://github.com/cleanlab/examples/blob/master/object_detection/detectron2_training.ipynb). Note that this tutorial just considers finding label issues in a subset of the COCO dataset (using held-out predictions for this subset). If you want to find all the mislabeled images across the entire COCO dataset, you can first execute this [other example notebook](https://github.com/cleanlab/examples/blob/master/object_detection/detectron2_training-kfold.ipynb) that uses K-fold cross-validation to produce out-of-sample predictions for every image, then use those labels and predictions in this tutorial.\n",
```